server.js

```diff
@@ -57,7 +57,7 @@ const authenticate = function(request, response, next) {
 
     let requestToken = request.body.token;
 
-    if (!requestToken) {
```

Comment:
this doesnt work, does it?

ok, you dont reach the first case. but yout still run in an exception in the very next case, since `requestToken !== config.globalConf.authToken` is true (null !== 'AUTH_DISABLED')

Reply:
Oops, went too fast on this there's also other issue with the request body, sorry for that.